src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invRobot = {};
+
+  for (const key in robot) {
+    if (invRobot.hasOwnProperty(robot[key])) {
```

Comment:
need create a variable for robot[key] 🙂

Reply:
Added the variable, thank you for the review 😄